pkg/factory/risingwave_object_factory.go

```diff
@@ -635,6 +655,8 @@ func (f *RisingWaveObjectFactory) coreEnvsForMeta(image string) []corev1.EnvVar
 		panic("unsupported meta storage type")
 	}
 
+	envVars = append(envVars, f.envsForLicense()...)
```

Comment:
Does the env apply to other components?

Reply:
No, it belongs to `MetaNodeOpts`

https://github.com/risingwavelabs/risingwave/blob/f54c7fe03198f5dd0c8a822763c380414290ce7f/src/meta/node/src/lib.rs#L191-L194